VSDiagnostics/VSDiagnostics/VSDiagnostics/Diagnostics/General/SwitchIsMissingDefaultLabel/SwitchIsMissingDefaultLabelCodeFix.cs

```diff
@@ -37,15 +37,31 @@ public override ImmutableArray<string> FixableDiagnosticIds
 
         private async Task<Document> AddDefaultCaseAsync(Document document, CompilationUnitSyntax root, SwitchStatementSyntax switchBlock)
         {
-            var notImplementedException =
-                SyntaxFactory.ThrowStatement(SyntaxFactory.ParseExpression("new System.NotImplementedException()"))
+            var model = await document.GetSemanticModelAsync();
+
+            var symbol = model.GetSymbolInfo(switchBlock.Expression).Symbol;
+            string expression;
+            if (symbol != null)
+            {
+                expression = $"nameof({symbol.Name})";
+            }
+            else if (switchBlock.Expression.IsKind(SyntaxKind.StringLiteralExpression))
```

Comment:
Instead of the elseif/if, can't you just do `else { expression = switchBlock.Expression.ToString(); }`?